### PR TITLE
pulltorefreshjs - Added missing function exports and properties to Options interface

### PR DESCRIPTION
**Pull request by humpedli**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://www.npmjs.com/package/pulltorefreshjs
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/pulltorefreshjs/index.d.ts [hunk 1]
```diff
@@ -1,10 +1,15 @@
-// Type definitions for pulltorefreshjs 0.1
+// Type definitions for pulltorefreshjs 0.2
```
Comment (DanielRosenwasser) [inline types/pulltorefreshjs/index.d.ts]: This should stay at `0.1`

File: types/pulltorefreshjs/index.d.ts [hunk 2]
```diff
@@ -108,6 +113,20 @@ export interface Options {
...
+     * Which condition should be met for pullToRefresh to trigger?
+     *
+     * Defaults to `!window.scrollY`
+     */
+    shouldPullToRefresh?(): boolean;
+
+    /**
+     * This value will be passed as `{ passive: true|false }` to `touchmove`
+     * listeners if passive-handlers are supported.
+     * (default: `false`)
```
Comment (DanielRosenwasser) [inline types/pulltorefreshjs/index.d.ts]: Can you keep this as "Defaults to `false`" to be consistent with the other comments?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]
Looks great, but the version number shouldn't be bumped since Definitely Typed packages are meant to align on major/minor numbers and are auto-incremented. pulltorefresh is still on 0.1.11, so we should stick to 0.1.

File: types/pulltorefreshjs/index.d.ts (hunk 2 above)
Comment (humpedli) [reply types/pulltorefreshjs/index.d.ts]: I changed this to the requested format, but there are other examples in this file for previous version too.

File: types/pulltorefreshjs/index.d.ts (hunk 1 above)
Comment (humpedli) [reply types/pulltorefreshjs/index.d.ts]: Fixed. I thought that I definitely have to increase this, but yes, I agree.

**Comment by DanielRosenwasser** [comment]
Thanks for the PR!

**Pull request merged**